sansa-examples-spark/src/main/scala/net/sansa_stack/examples/spark/query/GraphQuery.scala

```diff
@@ -0,0 +1,173 @@
+package net.sansa_stack.examples.spark.query
+
+import net.sansa_stack.query.spark.graph.jena.SparqlParser
+import net.sansa_stack.query.spark.graph.jena.model.{IntermediateResult, SparkExecutionModel, Config => modelConfig}
+import net.sansa_stack.rdf.spark.partition.graph.algo._
+import org.apache.jena.graph.Node
+import org.apache.jena.riot.Lang
+import org.apache.log4j.Logger
+import org.apache.spark.graphx.Graph
+
+import scala.concurrent.duration.Duration
+
+object GraphQuery {
+
+  def main(args: Array[String]): Unit = {
+
+    parser.parse(args, Config()) match {
+      case Some(config) => run(config)
+      case None => println(parser.usage)
+    }
+  }
+
+  def run(config: Config): Unit = {
+
+    println("===========================================")
+    println("| SANSA - Graph query example     |")
+    println("===========================================")
+
+    val log = Logger.getLogger(GraphQuery.getClass)
+
+    // set configures for query engine model
+    modelConfig.setAppName("SANSA Graph Query")
+      .setInputGraphFile(config.input)
+      .setInputQueryFile(config.query.head)
+      .setLang(Lang.NTRIPLES)
```

Comment:
These setting should be consistent to all the SANSA examples since we use the same for SANSA-Notebooks as well. But I guess we can change them soon.